socorro/cron/jobs/matviews.py

```diff
@@ -68,16 +68,6 @@ class SignaturesCronApp(_MatViewBackfillBase):
     depends_on = ('reports-clean',)
 
 
-class TCBSCronApp(_MatViewBackfillBase):
-    proc_name = 'update_tcbs'
```

Comment:
That will be part of the postgres simplification, yes. I'm going to write a list of all the database elements we can remove, and will proceed later on. I like to have migrations in their own, separate commits.